content/en/observability_pipelines/monitoring/metrics.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Metrics
+disable_toc: false
+aliases:
+- /path-to-old-doc/
```

Comment:
this needs to be updated or the section removed :)

Reply:
Removed